content/en/database_monitoring/setup_postgres/rds.md

```diff
@@ -245,6 +242,29 @@ To configure collecting Database Monitoring metrics for an Agent running on a ho
    ssl: allow
    ```
 
+If you want to authenticate with IAM, set the `region` and `instance_endpoint` parameters. **Note**: only set the `region` parameter if you want to use IAM authentication. IAM authentication takes precedence over the `password` field.
+
+   ```yaml
+   init_config:
+   instances:
+     - dbm: true
+       host: '<AWS_INSTANCE_ENDPOINT>'
+       port: 5432
+       username: datadog
+       aws:
+         instance_endpoint: '<AWS_INSTANCE_ENDPOINT>'
+         region: '<REGION>'
+       tags:
+         - "dbinstanceidentifier:<DB_INSTANCE_NAME>"
+       ## Required for Postgres 9.6: Uncomment these lines to use the functions created in the setup
+       # pg_stat_statements_view: datadog.pg_stat_statements()
+       # pg_stat_activity_view: datadog.pg_stat_activity()
+       ## Optional: Connect to a different database if needed for `custom_queries`
+       # dbname: '<DB_NAME>'
+   ```
+
+Please see [this guide][9] on how to configure IAM authentication on your RDS instance.
```

Comment:
any idea why this isnt rendering in the UI? (link number 9 is defined at the bottom & used elsewhere in this doc)

![Screen Shot 2023-11-15 at 10 14 58 AM](https://github.com/DataDog/documentation/assets/14317240/b299d7d2-4f05-4aec-a043-67e98b9c61ce)

Reply:
Yes, the link needs to go inside the tab. I pushed up a fix

Reply:
oh right of course, my bad